hphp/runtime/vm/jit/code-cache.cpp

```diff
@@ -97,7 +98,14 @@ CodeCache::CodeCache()
     exit(1);
```

Comment:
That code also exit()'s if code size is > 2GiB due to PC-relatives. Perhaps I marked the wrong exit(1)??? 

Anyway, I've also found that moving aHot around works well, but I'm afraid to do that since I don't know if __hot_start and __hot_end will be enabled, which could gain from having backward(to lower addresses) PC-relative branches from aHot into the __hot_start--__hot_end native functions.